site/content/blogs/release-v125.en.md

```diff
@@ -33,6 +33,39 @@ Copilot v1.25 brings several new features and improvements:
 ## Environment Addons
 ### Import Values From CloudFormation Stacks In Workload Manifests
 
+You can now import values from environment addons CloudFormation stack or any other arbitary stack in your workload manifest using `from_cfn`.
+To reference a value from another CloudFormation stack, users should first export the output value from the source stack.
```

Comment:
Can we have an example of what the `Export` should look like, and a link to CFN doc that talks about `Export`?

Reply:
added the example and link to doc